libcxx/utils/libcxx/test/features.py

```diff
@@ -355,6 +355,14 @@ def _mingwSupportsModules(cfg):
         name="has-no-zdump",
         when=lambda cfg: runScriptExitCode(cfg, ["zdump --version"]) != 0,
     ),
+    # Whether the `filecheck` executable is available. Note that this corresponds to
+    # a Python port of LLVM's FileCheck, not LLVM's actual FileCheck program, since
+    # that one requires building parts of LLVM that we don't want to build when merely
+    # testing libc++.
+    Feature(
+        name="has-filecheck",
+        when=lambda cfg: runScriptExitCode(cfg, ["filecheck --version"]) == 0,
+    ),
```

Comment:
I think allowing the external version is very reasonable, but IMO we should also look for `FileCheck`, since folks may very well have that already on their system (e.g. I do).

Reply:
The issue though is that we have to standardize on the case we're going to use in the tests. LLVM uses `FileCheck` and the Python port is `filecheck` (all lowercase).